Source/SvgFontManager.cs

```diff
@@ -0,0 +1,39 @@
+using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Drawing.Text;
+using System.IO;
+using System.Linq;
+using System.Text.RegularExpressions;
+
+namespace Svg
+{
+    public static class SvgFontManager
```

Comment:
Could you please add a description what this does and how it is used (along the lines you put in the comments in this PR already)?

Reply:
complete.